packages/xsnap/test/test-boot-lockdown.js

```diff
@@ -200,7 +206,6 @@ test('console - objects should include detail', async t => {
           'assertion text',
           '{"prop1":["elem1a","elem1b"],"prop2":["elem2a","elem2b"]}',
         ],
-        [],
```

Comment:
What do the two deleted empty arrays correspond to?

Reply:
`print()` calls where all the arguments got filtered. The lockdown warnings introduced several more, so I filtered them all out.

Reply:
p.s. I'm embarrased to explain this code. It clearly merits refactoring.